basics/didResolve.ts

```diff
@@ -0,0 +1,25 @@
+import { DidResolver } from '@kiltprotocol/sdk-js'
+import { Did, DidDocument } from '@kiltprotocol/types'
+
+export async function didResolve(did: Did): Promise<DidDocument | null> {
+  const { didResolutionMetadata, didDocument, didDocumentMetadata } =
+    await DidResolver.resolve(did)
+  if (didDocumentMetadata.deactivated) {
+    console.log(`DID ${did} Document is deactivated`)
+    return null
+  }
```

Comment:
Deactivated DIDs are associated with empty DID documents (with exception of an `id` property) by the resolver. Is there a reason you're changing this assumption?

Reply:
Yes, as per discussed updated.